.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -156,6 +156,7 @@ data class CIBuildModel(
                 TestCoverage(40, TestType.configCache, Os.LINUX, JvmCategory.MAX_VERSION, DEFAULT_LINUX_FUNCTIONAL_TEST_BUCKET_SIZE, arch = Arch.AMD64),
                 TestCoverage(41, TestType.configCache, Os.LINUX, JvmCategory.MAX_LTS_VERSION, DEFAULT_LINUX_FUNCTIONAL_TEST_BUCKET_SIZE, arch = Arch.AARCH64),
                 TestCoverage(42, TestType.configCache, Os.LINUX, JvmCategory.MAX_LTS_VERSION, DEFAULT_LINUX_FUNCTIONAL_TEST_BUCKET_SIZE, arch = Arch.AMD64),
+                TestCoverage(43, TestType.quick, Os.LINUX, JvmCategory.MAX_VERSION, expectedBucketNumber = DEFAULT_LINUX_FUNCTIONAL_TEST_BUCKET_SIZE)
```

Comment:
🤔❓ Looking at all the places we reference `Os.LINUX` I think adding `Os.ALPINE` would improve our test build setup.

This PR misses changes to `FunctionalTestBucketProvider.kt` in `GradleVersionRangeCrossVersionTestBucket::createFunctionalTestsFor`, and in `FunctionalTestBucketGenerator.kt` around `FunctionalTestBucketGenerator::splitBucketsByTestClassesForBuildProject` (in `parallelize` at `if (testCoverage.os == Os.LINUX)`).

`Os.ALPINE` would simplify all of them while only causing additional code in few (maybe two aside of the mentioned ones here) places.

Reply:
I don't like that the only thing connecting UUID 43 to Alpine is `parallelizationMethod.name = TestDistributionAlpine` in a file that is wayy to bad to review. `Enum.name` is also not the greatest thing to look for references ._.

Any better ideas?